index.js

```diff
@@ -5,7 +5,15 @@ const morseCodeDictionary = require("./morse-code-dictionary.json");
  * @param {String[]}  - An array of strings.
  * @returns {string[]} An array of strings.
  */
-function sortByStringLength() {}
+function sortByStringLength(input) {
+  const compare = (a,b) => {
```

Comment:
I like this but I'd actually prefer a shorter version of sort - remember we can use the default sort behavior so this could be written
`return input.sort((a,b) => { a.length - b.length })`
for the same result